lib/presentation/mixins/common_media_picker_mixin.dart

```diff
@@ -80,14 +82,36 @@ mixin CommonMediaPickerMixin {
       context,
       pickerConfig: onlyImage
           ? CameraPickerConfig(
+              textDelegate: getTextDelegateForLocale(
+                context,
+              ),
               enableAudio: false,
               onError: (e, a) => _onError(context: context, error: e),
             )
           : CameraPickerConfig(
+              textDelegate: getTextDelegateForLocale(
+                context,
+              ),
               enableRecording: true,
               onError: (e, a) => _onError(context: context, error: e),
             ),
-      locale: View.of(context).platformDispatcher.locale,
     );
   }
+
+  CameraPickerTextDelegate getTextDelegateForLocale(
+    BuildContext context,
+  ) {
+    switch (LocalizationService.currentLocale.value.languageCode) {
+      case 'ru':
+      case 'fr':
+        return LocalizedCameraPickerTextDelegate(
+          context,
+          LocalizationService.currentLocale.value.languageCode,
+        );
+      default:
+        return cameraPickerTextDelegateFromLocale(
+          LocalizationService.currentLocale.value,
+        );
+    }
+  }
```

Comment:
```suggestion
  CameraPickerTextDelegate getTextDelegateForLocale(
    BuildContext context,
  ) {
    return LocalizedCameraPickerTextDelegate(
          context,
          LocalizationService.currentLocale.value.languageCode,
        );
  }
```

No need to `switch`, WDYT?

Reply:
we get translations from L10n for Russian and French, but for Vietnamese and English, we use the translations provided by the wechat_camera package, which is why we need to switch between them